AzureCosmosDB/csharp/DocumentVectorPipelineFunctions/CosmosDBClientWrapper.cs

```diff
@@ -165,9 +164,6 @@ private class DocumentChunk
         [JsonPropertyName("id")]
         public string? ChunkId { get; init; }
 
-        [JsonPropertyName("page_number")]
```

Comment:
qq: we are planning to remove page number from output completely or come to it later?

Reply:
I'd love to keep it (I think it would be useful) but I'm not sure how to flow the information through the chunking effectively. Azure Doc Intelligence doesn't associate paragraphs with a page, and for .docx files, it seems to consider everything to be on the first page.